cmd/argo/commands/client/conn.go

```diff
@@ -19,7 +19,10 @@ var (
 
 var overrides = clientcmd.ConfigOverrides{}
 
-var explicitPath string
+var (
+	explicitPath string
+	Offline      bool
```

Comment:
I wonder if we should have `ARGO_OFFLINE` envvar?

Reply:
I think it would be better suited as an argument atm since it is only supported for the linting.